tests/core/test_task.py

```diff
@@ -48,6 +48,8 @@ def output(self):
         self.assertEqual(task._get_output_target("file_b").path, task.get_output_file_name("file_b"))
         self.assertIn("file_b", task.get_output_file_name("file_b"))
         self.assertIn("some_parameter=42", task.get_output_file_name("file_b"))
+        self.assertIn("some_parameter=42/file_a", list(task.get_all_input_file_names())[0])
```

Comment:
Please also add a test, that the result has length 1.

Reply:
I am not sure if I understand correctly. Why do we care at this place that the result has length 1? Several files could be required by the test task and for us it would be fine at least the name of the fist item in the list is that one we expect.

Reply:
In the test case, we know exactly what should the output be - not more and not less.
(I did not check if it should be 1, but it should definitely be a single number - deterministic).